package.json

```diff
@@ -6,6 +6,7 @@
   "description": "MedMorph Backend Service Client",
   "scripts": {
     "start": "npm-run-all --parallel startFrontend startBackend",
+    "startProd": "WATCH=false npm-run-all -s startFrontend startBackend",
```

Comment:
Even though `start` works just fine, the frontend takes a bit to build and the backend starts up before it, so you get a 404 when you try to load the frontend page as it tries to find a `bundle.js` file that doesn't exist yet.

I don't think having webpack watch the files, or nodemon watch the backend files, really makes much of a difference, but I thought it would be nice to have them run sequentially (-s) instead of in parallel.

Normally it runs in parallel because the `watch` part of webpack causes the job to persist and you'd never get to the `startBackend` in sequence.  But if we don't care about watching the react files, we can just have webpack build it out and not watch, which means we can sequentially run the two.

In short, I'm looking for opinions on what we should do here.  I don't think it really matters, but perhaps it has some implications on load time I'm not thinking about.  Our options are to run it like this - building out the frontend bundle and then running the express server - or to just do a simple `npm start` so the docker container runs them in parallel, and watches the files even though they won't change.

Reply:
Can we have webpack build the "production build" like `npm build` traditionally does? (Is there any difference between that and what we're doing now?)

Reply:
the only difference is the `watch` property.  Traditionally `npm build` would make a build folder with `bundle.js`, which webpack does every time regardless of whether its production or development. If the `watch` property is set to `false` it is almost identical, it builds out the react files.